emails/models.py

```diff
@@ -735,6 +734,10 @@ def profile(self):
     def owner_has_premium(self):
         return self.profile.has_premium
 
+    @property
+    def owner_has_phone(self):
+        return self.profile.has_phone
+
```

Comment:
question: Does `reply` need an `owner_has_phone` property? I don't see it used anywhere.

Reply:
Removed in e4f493fe9123f844f733797911f1c01547b82ff2